### PR TITLE
render header and footer directly as DOM element target

### PR DESCRIPTION
**Pull request by fuentesloic**
As the Header and Footer component are indepentdants and could not move during this projet, I tried to let them alone, far, ... separate.
It's just a guess/try but it should works (or not ...). Let me know!

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -6,13 +6,13 @@ import 'typeface-roboto-condensed';
...
 import * as serviceWorker from './serviceWorker';
-import Header from '../components/Layout/Header';
-import Footer from '../components/Layout/Footer';
+import Header from './components/Layout/Header';
+import Footer from './components/Layout/Footer';
 import AppRouter from './routes/AppRouter';
 
-ReactDOM.render(<Header />, document.getElementById('Header'));
+ReactDOM.render(<Header />, document.getElementById('header'));
 ReactDOM.render(<AppRouter />, document.getElementById('Main'));
```
Comment (fuentesloic) [inline src/index.js]: Should we also change `document.getElementById('Main'));`? from `Main` to `main`?

**Comment by aurelienroux** [comment]
@fuentesloic not working as we have Links in Header that are not included in AppRouter, crashing the app. guess Header and Footer components will always have to be included in Router, as they contribute to routing

`Error: Invariant failed: You should not use <Link> outside a <Router>`

File: src/index.js [hunk 2]
```diff
@@ -6,13 +6,13 @@ import 'typeface-roboto-condensed';
 import 'typeface-montserrat';
 import './styles/styles.scss';
 import * as serviceWorker from './serviceWorker';
-import Header from '../components/Layout/Header';
-import Footer from '../components/Layout/Footer';
```
Comment (fuentesloic) [inline src/index.js]: to avoid this kind of error, you could use alias (it will change your life ^^)

**Comment by fuentesloic** [comment]
> @fuentesloic not working as we have Links in Header that are not included in AppRouter, crashing the app. guess Header and Footer components will always have to be included in Router, as they contribute to routing
> 
> `Error: Invariant failed: You should not use <Link> outside a <Router>`

Arf ... boring :(

> @fuentesloic not working as we have Links in Header that are not included in AppRouter, crashing the app. guess Header and Footer components will always have to be included in Router, as they contribute to routing
> 
> `Error: Invariant failed: You should not use <Link> outside a <Router>`

Arf ... boring 🤕

File: src/index.js (hunk 1 above)
Comment (aurelienroux) [reply src/index.js]: good point!

**Pull request closed without merging**